debian/yaru-theme-mate-abandon-Yaru-MATE-light-and-Yaru-MATE-dark.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+yaru_mate_light_name="'Yaru-MATE-light'"
+yaru_mate_dark_name="'Yaru-MATE-dark'"
+yaru_mate_new_name="'Yaru-mate'"
+yaru_mate_dark_new_name="'Yaru-mate-dark'"
+yaru_cursor_name="'Yaru'"
+
+if [ "$(gsettings get org.mate.desktop.interface gtk-theme)" = "$yaru_mate_light_name" ]; then
+    gsettings set org.mate.desktop.interface gtk-theme "$yaru_mate_new_name"
+fi
```

Comment:
Shouldn't we also migrate the old `Yaru-MATE` gtk theme to `Yaru-mate`?
Btw, will the old `Yaru-MATE` folders be automatically removed, or should we remove them manually?

Reply:
> Shouldn't we also migrate the old Yaru-MATE gtk theme to Yaru-mate?

ubuntu-MATE has been using only `Yaru-MATE-light` and `Yaru-MATE-dark` as it was decided to not include the previously default inverted headerbar `Yaru-MATE` theme from the beginning.

> Btw, will the old Yaru-MATE folders be automatically removed, or should we remove them manually?

we don't need to remove anything if [`ubuntu-mate-artwork`](https://github.com/ubuntu-mate/ubuntu-mate-artwork/tree/master/usr/share/themes) removes the packaging for `Yaru-MATE` themes  :)

Reply:
Okay, thanks :) :+1: